frontend/src/util/validations.js

```diff
@@ -0,0 +1,19 @@
+const _ = require("lodash");
+
+function languageValidator(language) {
+  return _.includes(["hebrew", "english"], language);
```

Comment:
Nice! the only addition is I would bring the array of supported languages from the BE and save it in the global context